core/src/test/java/org/infinispan/functional/AbstractFunctionalTest.java

```diff
@@ -43,8 +48,15 @@ protected void createCacheManagers() throws Throwable {
       replBuilder.clustering().cacheMode(isSync ? CacheMode.REPL_SYNC : CacheMode.REPL_ASYNC);
       configureCache(replBuilder);
       cacheManagers.stream().forEach(cm -> cm.defineConfiguration(REPL, replBuilder.build()));
+      // Create scattered caches
+      if (!Boolean.TRUE.equals(transactional)) {
```

Comment:
I can init the `transactional` to false in the constructor... I think there was some reason to have this as a tri-state, maybe related to `@InTransactionalMode`.